src/features/todos.ts

```diff
@@ -1,8 +1,21 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSelector, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
+import { RootState } from '../app/store';
 
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    set: (_, action) => {
+      return action.payload;
```

Comment:
Consider explicitly typing the `action` parameter in the `set` reducer to ensure type safety. This can be done by specifying the type as `PayloadAction<Todo[]>` to match the expected payload structure.